shardingsphere-sql-parser/shardingsphere-sql-parser-mysql/src/main/antlr4/imports/mysql/DALStatement.g4

```diff
@@ -84,7 +84,11 @@ showProfileType
     ;
 
 setVariable
-    : SET variable?
+    : SET (variable? (TO | EQ_) variableValue COMMA_?)*
```

Comment:
The rule is `SET variable = expr [, variable = expr] ...`
This rule is not accurate, for example, if only one variable and variableValue, the COMMA_ should never exist.
Where is the definition of keyword `TO` ?

Reply:
done